build/msw/wx_setup.props

```diff
@@ -119,5 +120,11 @@
     <BuildMacro Include="wxIntRootDir">
       <Value>$(wxIntRootDir)</Value>
     </BuildMacro>
+    <BuildMacro Include="wxReleaseRuntimeLibrary">
+      <Value>$(wxReleaseRuntimeLibrary)</Value>
+    </BuildMacro>
+    <BuildMacro Include="wxDebugRuntimeLibrary">
+      <Value>$(wxDebugRuntimeLibrary)</Value>
+    </BuildMacro>
```

Comment:
This seems to be unused?

Reply:
Ah, I understand better! I'll change to that (which will make these used)

Reply:
Modified to use wxReleaseRuntimeLibrary/wxDebugRuntimeLibrary in vcxproj files (the original way), plus use wxRuntimeLibs in wx_setup to set initial values. Default is /MD. Setting wxRuntimeLibs to 'static' will use /MT (any other value will leave the setting at /MD)